core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.html

```diff
@@ -1,81 +1,216 @@
+      <h3>Subtopics</h3>
+      <button class="btn add-subtopic-btn protractor-test-add-subtopic-button" ng-click="createSubtopic()">
+        + ADD SUBTOPIC
       </button>
-      <stories-list ng-if="topic.getCanonicalStoryIds().length > 0"
-                    story-summaries="canonicalStorySummaries"
-                    topic="topic">
-      </stories-list>
-      <h4 ng-if="topic.getCanonicalStoryIds().length === 0" style="color: gray;">
-        Click on Create Story to create a new canonical story for the topic
+      <h4 ng-if="subtopics.length === 0" class="add-entity-text">
+        Click on Add Subtopic to create a new subtopic for the topic.
       </h4>
-    </div>
-  </md-card>
+      <div ng-if="subtopics.length">
+        <div class="list-header">
+          <span>Name</span>
+          <span># of Skills</span>
+        </div>
+        <div ng-repeat="subtopic in subtopics">
+          <div class="subtopics-list-item">
+            <span class="protractor-test-subtopic" ng-click="navigateToSubtopic(subtopic.getId())"><[subtopic.getTitle()]></span>
+            <span><[subtopic.getSkillSummaries().length]>
+              <button class="btn btn-secondary protractor-test-delete-subtopic-button" ng-click="deleteSubtopic(subtopic.getId())">Delete</button>
+            </span>
+          </div>
+        </div>
+      </div>
+    </md-card>
+    <md-card class="oppia-subtopics-list-card">
+      <h3>Skills</h3>
+      <button class="btn add-subtopic-btn" ng-click="createSkill()">
+        + ADD SKILL
+      </button>
+      <h4 ng-if="uncategorizedSkillSummaries.length === 0" class="add-entity-text">
+        Click on Add Skill to create a new skill for the topic.
+      </h4>
+      <div ng-if="uncategorizedSkillSummaries.length">
+        <div class="list-header">
+          <span>Name</span>
+        </div>
+        <div ng-repeat="skillSummary in uncategorizedSkillSummaries" class="skills-list-item">
+          <a ng-href="<[getSkillEditorUrl(skillSummary.getId())]>" target="_blank" rel="noopener">
+            <span><[skillSummary.getDescription()]></span>
+          </a>
+          <button class="btn btn-secondary"
+                  ng-click="deleteUncategorizedSkillFromTopic(skillSummary)">
+            Delete
+          </button>
+        </div>
+      </div>
+    </md-card>
+  </div>
+  <div class="topic-editor-right">
+    <md-card class="oppia-subtopics-list-card">
+      <h3> Canonical Stories </h3>
+      <button class="btn add-subtopic-btn protractor-test-create-story-button"
+              ng-click="createCanonicalStory()">
+        + ADD STORY
+      </button>
+      <div class="canonical-stories">
+        <stories-list ng-if="topic.getCanonicalStoryIds().length > 0"
+                      story-summaries="canonicalStorySummaries"
+                      topic="topic">
+        </stories-list>
+        <h4 ng-if="topic.getCanonicalStoryIds().length === 0" style="color: gray;">
```

Comment:
Maybe use ones of [these](https://getbootstrap.com/docs/4.0/utilities/colors/) instead of inline style?

Reply:
That's a lot better. Done